src/components/swap/AdvancedSwapDetails.tsx

```diff
@@ -176,3 +181,26 @@ export function AdvancedSwapDetails({ trade, allowedSlippage, syncing = false }:
     </Column>
   )
 }
+
+function TokenTaxLineItem({ trade, type }: { trade: ClassicTrade; type: 'input' | 'output' }) {
+  const [currency, percentage] =
+    type === 'input' ? [trade.inputAmount.currency, trade.inputTax] : [trade.outputAmount.currency, trade.outputTax]
+
+  if (percentage.equalTo(ZERO_PERCENT)) return null
+
+  return (
+    <RowBetween>
+      <MouseoverTooltip
+        text={
+          <Trans>
+            Some tokens take a fee when they are bought or sold, which is set by the token issuer. Uniswap does not
+            receive any share of these fees.
+          </Trans>
+        }
+      >
+        <ThemedText.BodySmall color="textSecondary">{`${currency.symbol} fee`}</ThemedText.BodySmall>
```

Comment:
Confusing (and *non-standard* way of expressing value - the currency symbol should be adjacent to the number, *not* separated by the word "fee".

Reply:
this is the label for the token fee though

Reply:
@zzmp this is a percentage line item, not a currency amount:
<img width="387" alt="image" src="https://github.com/Uniswap/interface/assets/39385577/53f15187-cb90-4ae2-8fcb-7efb935e5f05">